src/interpreter/plugin/DatePlugin.ts

```diff
@@ -45,23 +78,96 @@ export class DatePlugin extends FunctionPlugin {
     'EOMONTH': {
       method: 'eomonth',
       parameters: [
-          {argumentType: ArgumentTypes.NUMBER},
+          {argumentType: ArgumentTypes.NUMBER, minValue: 0},
           {argumentType: ArgumentTypes.NUMBER},
         ]
     },
     'DAY': {
       method: 'day',
-      parameters: [
-          {argumentType: ArgumentTypes.NUMBER},
+      parameters:  [
+          {argumentType: ArgumentTypes.NUMBER, minValue: 0},
         ]
     },
     'DAYS': {
       method: 'days',
-      parameters:[
-          {argumentType: ArgumentTypes.NUMBER},
-          {argumentType: ArgumentTypes.NUMBER},
+      parameters: [
+          {argumentType: ArgumentTypes.NUMBER, minValue: 0},
```

Comment:
Why minValue: 0?

Reply:
XL compatibility

GS allows for negative values

Reply:
Good point! So we can mention that in the corresponding tests that we have:
`// Inconsistency with Product `
Do I understand correctly, that in such cases this would be safe to change (simply by removing the constraint on the argument) ?

Reply:
yes